Ml.net

BillWagner opened the pull request:
Publish ml.net documentation

Note: This PR should not be squashed to preserve original authors.

mairaw committed:
config changes for docs-internal only

aditidugar-zz committed:
Create ML folder and add Overview

mairaw committed:
Merge remote-tracking branch 'upstream/master' into ml.net

aditidugar-zz committed:
Adding glossary file

mairaw committed:
Merge remote-tracking branch 'upstream/master'

mairaw committed:
Merge branch 'ml.net' into ml.net

aditidugar-zz committed:
Remove breaks, spans, and fix folder name

aditidugar-zz committed:
Remove span in index.md

JRAlexander committed:
Merge pull request #69 from aditidugar/ml.net

Add ML.NET folder

Commit:
Created placeholder files and folder for tutorials and resource folder

Commit:
Revised based on review and added index file to resources folder

Commit:
Fixed title

JRAlexander committed:
Merge pull request #70 from JRAlexander/mlnet

Created placeholder files and folder for tutorials and resource folder

mairaw committed:
add ML.NET items to the breadcrumb (#71)

Commit:
update toc  for ml.net

JRAlexander committed:
Merge pull request #72 from JRAlexander/mlnet2

Update Table of Contents for ML.NET

Commit:
added ml.net to hub and added landing page w images

Commit:
Revised based on review

Commit:
Revised to test images

Commit:
updating images

Commit:
updated to test

Commit:
revised based on feedback

JRAlexander committed:
Merge pull request #73 from JRAlexander/mlnet-landing

Added ml.net to hub and added landing page with images

aditidugar-zz committed:
Updates to overview pages and file name changes

aditidugar-zz committed:
Address PR comments

JRAlexander committed:
Merge pull request #74 from dotnet/ml

Updates to overview pages and file name changes

mairaw committed:
Merge remote-tracking branch 'upstream/master'

mairaw committed:
Merge remote-tracking branch 'upstream/master' into ml.net

JRAlexander committed:
ML.NET Sentiment Analysis Walkthrough (#75)

* initial commit

* Added ml basics, updated glossary,  and reworked tutorial

* revised ml.net toc

* fixed broken links

* Removing branch links

mairaw committed:
global setting for ML (#76)

mairaw committed:
Merge remote-tracking branch 'upstream/master'

mairaw committed:
Merge branch 'master' into ml.net

aditidugar committed:
Updates to glossary.md (#77)

* Modify glossary terms based on team feedback

* Fix Acrolinx suggestions

* update per feedback

Update based on Maira and my feedback.

Note that some questions remain.

* few more edits

mairaw committed:
Resources updates (#80)

* rename file

* updates

* Update basics.md

remove extra '/' in URLs

mairaw committed:
Metadata and sentiment analysis tutorial updates (#78)

* updates

* old feedback from Bill

* added links

* added blog post title

aditidugar committed:
Taxitrip (#79)

* Add first draft of taxi-fare tutorial

* fix TOC

* fix link on tutorials index

* update link

* Fix based on feedback and add start of Evaluation and Prediction sections

* edit pass

more 1st person in the explanatory text.
Lots of passive voice.
Lots of future tense.

* await the async call

Once that call is a awaited, that causes some ripple effects in the sample code.
The anchor links to the glossary was mispelled.

* wrong folder in link

BillWagner committed:
Publish ml.net

BillWagner committed:
restore doc set

use the public docset for the public repo.

mairaw committed:
remove internal path

BillWagner committed:
Fix xref errors

TextLoader needs %601 Zip needs %2A

pkulikov commented on docs/machine-learning/index.md:
```diff
@@ -0,0 +1,33 @@
+
+ML.NET is a free, open-source, and cross-platform machine learning framework that enables you to build custom machine learning solutions and integrate them into your .NET applications. This guide provides many resources about working with ML.NET.
+
+For more information about ML.NET, see [ntroducing ML.NET: Cross-platform, Proven and Open Source Machine Learning Framework ](https://blogs.msdn.microsoft.com/dotnet/) post on the .NET blog.
```
Typo in the link text: *ntroducing ML.NET...*
In general, that is great, great addition!!!

BillWagner committed:
fix type

And, code fence BinaryClassificationMetrics because the xref isn't working now.

mairaw committed:
removed extra space

aditidugar-zz committed:
Update taxi far tutorial with correct links and data paths

BillWagner committed:
add hub image

mairaw committed:
updating hub image + add api link

BillWagner committed:
restore the xref.

The new API reference has built correctly.

BillWagner committed:
Merge branch 'ml.net' of https://github.com/BillWagner/docs into ml.net

mairaw reviewed (approved):
woo hoo! 🎆
